scripts/data_prep/convert_delta_to_json.py

```diff
@@ -0,0 +1,451 @@
+            res = w.clusters.get(
+                cluster_id=compute_id)  # '0704-124501-tsc2fxq' - invalid 12.2.x
+            runtime_version = res.spark_version.split('-scala')[0].replace(
+                'x-snapshot', '0').replace('x', '0')
+            if version.parse(runtime_version) < version.parse(
+                    MINIMUM_DBR_VERSION):
+                raise RuntimeError(
+                    f'You need at least {MINIMUM_DBR_VERSION} to use Databricks-connect to read delta table for FT API but got {res.spark_version}'
+                )
+            sparkSession = DatabricksSession.builder.remote(
+                host=args.DATABRICKS_HOST,
+                token=args.DATABRICKS_TOKEN,
+                cluster_id=compute_id).getOrCreate()
+
+    fetch(method, args.delta_table_name, args.json_output_path, args.batch_size,
+          args.processes, sparkSession, dbsql)
+
+    if dbsql is not None:
+        dbsql.close()
+
+    iterative_combine_jsons(
+        args.json_output_path,
+        os.path.join(args.json_output_path, 'combined.jsonl'))
+
+
+if __name__ == '__main__':
+    parser = ArgumentParser(
+        description=
+        'Download delta table from UC and convert to json to save local')
+    parser.add_argument('--delta_table_name',
+                        required=True,
+                        type=str,
+                        help='UC table <catalog>.<schema>.<table name>')
+    parser.add_argument('--json_output_path',
+                        required=True,
+                        type=str,
+                        help='Local path to save the converted json')
+    parser.add_argument('--DATABRICKS_HOST',
+                        required=False,
+                        type=str,
+                        help='DATABRICKS_HOST')
+    parser.add_argument('--DATABRICKS_TOKEN',
+                        required=False,
+                        type=str,
+                        help='DATABRICKS_TOKEN')
+    parser.add_argument('--http_path',
+                        required=False,
+                        type=str,
+                        help='http_path is set then dbsql method is used')
+    parser.add_argument('--batch_size',
+                        required=False,
+                        type=int,
+                        default=1 << 30,
+                        help='row chunks to transmit a time to avoid OOM')
+    parser.add_argument('--processes',
+                        required=False,
+                        type=int,
+                        default=1,
+                        help='number of processes allowed to use')
+    parser.add_argument(
+        '--cluster_id',
+        required=True,
+        type=str,
+        default=None,
+        help=
+        'Use serverless if not present. IMPORTANT! make sure cluster has runtime newer than 14.1.0, the databricks-connect client version'
+    )
+    parser.add_argument('--debug', type=bool, required=False, default=False)
```

Comment:
looks like you are utilizing this arg in the code. Do you have a plan to add some code when user enable `debug` mode?

Reply:
Good catch. ``debug`` is removed. This was from early development (used by myself).